### PR TITLE
OSOE-611: Update documentation on global config files

### LINKED ISSUE #44: NodeJsExtensionsGlobalESLintConfigurationDirectory and NodeJsExtensionsGlobalStylelintConfigurationDirectory only works with a submodule (OSOE-611)
Adding the following to the _Directory.Build.props_ file in a solution where a project uses NE from NuGet...

```xml
  <PropertyGroup>
    <!-- Instruct Node.js Extensions to create .eslintrc and .stylelintrc files in the current directory, which is the
         solution root directory. -->
    <NodeJsExtensionsGlobalESLintConfigurationDirectory>$(MSBuildThisFileDirectory)</NodeJsExtensionsGlobalESLintConfigurationDirectory>
    <NodeJsExtensionsGlobalStylelintConfigurationDirectory>$(MSBuildThisFileDirectory)</NodeJsExtensionsGlobalStylelintConfigurationDirectory>
  </PropertyGroup>
```

...yields build errors, and e.g. the following during SCSS build:

> no-path(1,1): error ERROR: No configuration provided for ...\site.scss.

The NuGet sample actually indeed does the opposite, and local config files work.

I have a faint memory of this being an acceptable disadvantage that we couldn't somehow work around. However, I'd expect this to be documented under the docs pages where the above configs are mentioned.

[Jira issue](https://lombiq.atlassian.net/browse/OSOE-611)

### PR DESCRIPTION
**Pull request by 0liver**
[OSOE-611](https://lombiq.atlassian.net/browse/OSOE-611)
Fixes #44

[OSOE-611]: https://lombiq.atlassian.net/browse/OSOE-611?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: Lombiq.NodeJs.Extensions/Docs/Styles.md [hunk 1]
```diff
@@ -65,16 +65,20 @@ The `build:styles` script is a wrapper to execute the `lint:styles` and `compile
...
+
+> ℹ This option only works when using Node.js Extensions from a submodule, **not** from the NuGet package.
+
+In order to use a global _.stylelintrc.js_ file for your whole solution, you can instruct Node.js Extensions to create that file in the location specified by the MSBuild property `<NodeJsExtensionsGlobalStylelintConfigurationDirectory>`. This property is easiest to add in a _Directory.Build.props_ file in your solution's root directory as follows:
 
 ```xml
 <NodeJsExtensionsGlobalStylelintConfigurationDirectory>$(MSBuildThisFileDirectory)</NodeJsExtensionsGlobalStylelintConfigurationDirectory>
 ```
 
-> ℹ Please edit _.stylelintrc.js_ once it has been created, and adjust the path to _.stylelintrc.lombiq-base.js_ according to your solution's directory structure.
```
Comment (Piedone) [inline Lombiq.NodeJs.Extensions/Docs/Styles.md:76]: Is this not needed anymore?

File: Lombiq.NodeJs.Extensions/Docs/JavaScript.md [hunk 2]
```diff
@@ -81,25 +85,7 @@ If a certain rule's violation is incorrect in a given location, or you want to s
...
-
-1. `<NodeJsExtensionsGlobalESLintConfigurationDirectory>` is set to a valid directory path.
-2. If a _package.json_ file does not exist at that path yet, Node.js Extensions will create it (but only when using Node.js Extensions from a submodule).
-3. Building your solution will install the necessary dependencies.
-
-Afterwards, Visual Studio will show ESLint warnings already during development, using the same configuration that will be used during the build.
-
-#### Maintenance
-
-In order to keep the ESLint rules and configuration in-sync between Visual Studio and Node.js Extensions, make sure all of the package dependencies in your _package.json_ starting with `eslint` always use the same versions as those in Node.js Extensions' _package.json_, especially after updating the latter.
```
Comment (Piedone) [inline Lombiq.NodeJs.Extensions/Docs/JavaScript.md:102]: Is this not needed anymore?

Comment (0liver) [reply Lombiq.NodeJs.Extensions/Docs/JavaScript.md:102]: No. We update those automatically from NE.

File: Lombiq.NodeJs.Extensions/Docs/Styles.md (hunk 1 above)
Comment (0liver) [reply Lombiq.NodeJs.Extensions/Docs/Styles.md:76]: I've moved this up a few lines: https://github.com/Lombiq/NodeJs-Extensions/pull/56/files/ffffe57f2ad93183dc20c156dd52a82ab6d5fd7d#diff-33acbba1e704c6e4f93652f05a1b55652325239733cf57e1c33da74bba369307R70.

**Pull request merged**